graphql/queries.ts

```diff
@@ -76,6 +76,42 @@ export const GET_SUBMISSIONS = gql`
   }
 `
 
+export const ACCEPT_SUBMISSION = gql`
+  mutation acceptSubmission(
+    $submissionId: String!
+    $comment: String!
+    $reviewer: Int!
```

Comment:
could `reviewer` just be taken from the user object in the request? This way we don't introduce a new security where a user can pretend to be anyone in their reviews.

Reply:
What do you mean by the user object in the request?

Reply:
I'm grabbing it from the resolver ctx now 👍